templates/systemd/funkwhale-api.service.j2

```diff
@@ -26,6 +26,7 @@ ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} create \
                 --tmpfs=/tmp:rw,noexec,nosuid,size={{ funkwhale_api_tmpfs_size }} \
                 --read-only \
                 --env-file={{ funkwhale_api_env_path }} \
+                {{ funkwhale_api_process_extra_arguments | join(' ') }} \
```

Comment:
Your variables are named `_process_extra_arguments`, which implies that these arguments are passed to the process.

For some of these instances, it's true - the arguments are indeed passed to the process.

For others (e.g. `funkwhale_api_process_extra_arguments`), the arguments are passed to the `docker run` statement before the container image name, so these are actually "container (run) extra arguments". The variable should probably be renamed from `funkwhale_api_process_extra_arguments` to `funkwhale_api_container_extra_arguments` (naming which is consitent with other roles - e.g. `nextcloud_container_extra_arguments` used by the [Nextcloud role](https://github.com/mother-of-all-self-hosting/ansible-role-nextcloud), etc.)

----

Of course, if you need to pass arguments to both the container and to the process, then you'd need both kinds of variables (e.g. `funkwhale_api_container_extra_arguments` and `funkwhale_api_process_extra_arguments`).

Reply:
The `devture_traefik` role adds uses both `_container_` and ``_process_` variables, but adds an additional conditional, would that be desirable for this PR to use a similar syntax?

```
{% if devture_traefik_container_extra_arguments | length > 0 %}
{{ devture_traefik_container_extra_arguments | join(" ") }} \
{% endif %}
{{ devture_traefik_container_image }} \
--configFile=/config/traefik.yml {{ devture_traefik_process_extra_arguments | join(' ') }}

```